source/features/tab-size.css

```diff
@@ -1,6 +1,6 @@
 /* Make tab-indented code more readable on GitHub and Gist */
 
-* {
+.markdown-body {
```

Comment:
In this case I don't think the rest of the file is needed, it needs to be reviewed

Reply:
Maybe more specifically `code` because I don’t think tabs are rendered anywhere else in markdown
```suggestion
.markdown-body code {
```

Reply:
> In this case I don't think the rest of the file is needed, it needs to be reviewed

It's for `show-whitespaces`

> Maybe more specifically `code` because I don’t think tabs are rendered anywhere else in markdown

`code` is for inline code. `pre` isn't right either, we need to use [`codeElementsSelector`](https://github.com/refined-github/refined-github/blob/1c04d21e0f2391c9249157919b4db62847422a3b/source/github-helpers/dom-formatters.ts#L14), though you can see how this is unnecessary

Reply:
`Code` isn't for inline code, every `pre` usually *also* wraps a `code`. Either way pre or code should be good enough here. 

> It's for show-whitespaces

If it only affects that now, we should move it there (and also extend it because there are 12 steps, not just 2,4,8,10,13)

Reply:
Anyway not a big deal, it can stay without `code`

Reply:
> `Code` isn't for inline code, every `pre` usually _also_ wraps a `code`

I'm talking about how GitHub uses it, not HTML semantics

> If it only affects that now, we should move it there (and also extend it because there are 12 steps, not just 2,4,8,10,13)

You just read my mind 😄

Reply:
> I'm talking about how GitHub uses it

Yeah, same:

```
here
```

<img width="432" alt="Screenshot 7" src="https://user-images.githubusercontent.com/1402241/230772018-b95cccc5-ed5e-4426-bab0-27bfc7d195cf.png">

> `code` is for inline code

So, again, `code` is _not_ for "inline code", it's just for "code"